src/v3/test/integration/__snapshots__/enroll-profile-new.test.tsx.snap

```diff
@@ -54,29 +54,43 @@ exports[`enroll-profile-new should render form 1`] = `
                 class="MuiBox-root emotion-7"
               >
                 <div
-                  class="MuiBox-root emotion-11"
+                  class="MuiBox-root emotion-8"
+                >
+                  <p
+                    class="MuiTypography-root MuiTypography-body1 MuiTypography-paragraph emotion-12"
+                    data-se="o-form-explain"
+                    id="enroll-profile_Description_Fields_are_required_unless_marked_optional_2"
+                  >
+                    Fields are required unless marked optional.
+                  </p>
+                </div>
+              </div>
+              <div
+                class="MuiBox-root emotion-7"
+              >
+                <div
+                  class="MuiBox-root emotion-14"
                 >
                   <label
-                    class="MuiFormLabel-root MuiInputLabel-root MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root emotion-12"
+                    class="MuiFormLabel-root MuiInputLabel-root MuiFormLabel-colorPrimary MuiInputLabel-root emotion-15"
                     for="userProfile.lastName"
                   >
                     Last name
                     <span
                       aria-hidden="true"
-                      class="MuiFormLabel-asterisk MuiInputLabel-asterisk emotion-13"
+                      class="no-translate MuiBox-root emotion-14"
                     >
-                       
-                      *
+                       *
                     </span>
                   </label>
                   <div
-                    class="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth emotion-14"
+                    class="MuiInputBase-root MuiInputBase-colorPrimary MuiInputBase-fullWidth emotion-17"
```

Comment:
what is this change in `emotion` from?

Reply:
This happened because a new [subtitle](https://github.com/okta/okta-signin-widget/pull/3205/files#diff-4a73bc5dbaad590a7b9422758173375c1db48abbcd3cceed5370ec963f54f381R59-R66) was added to this view which forced an increase in the total number of elements which causes the emotion element count to increase.